### PR TITLE
chore(ci): add support for nightly acceptance tests

### PR DESCRIPTION
**Pull request by remyleone**
Before merging that, we should be sure that no credentials are shown while running it. Especially while we are using the old SDK.

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          fetch-depth: 2
+      - name: Acceptance Tests
+        run: go test -v ./... -timeout=1h -run TestAccScaleway${{ matrix.products }}*
+        env:
+          TF_LOG: DEBUG
+          TF_ACC: 1
+          SCW_DEBUG: 1
+          SCW_SECRET_KEY: ${{ secrets.SCW_SECRET_KEY }}
+          SCW_DEFAULT_ORGANIZATION_ID: ${{ secrets.SCW_DEFAULT_ORGANIZATION_ID }}
+          SCW_ACCESS_KEY: ${{ secrets.SCW_ACCESS_KEY }}
```
Comment (QuentinBrosse) [inline .github/workflows/nightly.yml:39]: nit
```suggestion
          SCW_ACCESS_KEY: ${{ secrets.SCW_ACCESS_KEY }}
          SCW_SECRET_KEY: ${{ secrets.SCW_SECRET_KEY }}
          SCW_DEFAULT_ORGANIZATION_ID: ${{ secrets.SCW_DEFAULT_ORGANIZATION_ID }}
```

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.1
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Acceptance Tests
```
Comment (QuentinBrosse) [inline .github/workflows/nightly.yml:31]: nit
```suggestion
      - name: Run Acceptance Tests
```

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: nightly acceptance tests
```
Comment (QuentinBrosse) [inline .github/workflows/nightly.yml:1]: nit
```suggestion
name: Nightly Acceptance Tests
```

**Review by QuentinBrosse — APPROVED** [review]
Some NITs

**Review by QuentinBrosse — APPROVED** [review]
LGTM

File: .github/workflows/nightly.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.1
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```
Comment (jerome-quere) [inline .github/workflows/nightly.yml:30]: why 2 ?

File: scaleway/resource_lb_ip_beta.go [hunk 5]
```diff
@@ -80,12 +80,30 @@ func resourceScalewayLbIPBetaRead(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
-	_ = d.Set("region", string(region))
-	_ = d.Set("organization_id", res.OrganizationID)
-	_ = d.Set("ip_id", res.ID)
-	_ = d.Set("ip_address", res.IPAddress)
-	_ = d.Set("reverse", res.Reverse)
-	_ = d.Set("lb_ip", flattenStringPtr(res.LBID))
+	err = d.Set("region", string(region))
```
Comment (jerome-quere) [inline scaleway/resource_lb_ip_beta.go:83]: We ignore `Set` error everywhere else why change here ?

**Review by jerome-quere — CHANGES_REQUESTED** [review]

File: scaleway/resource_lb_certificate_beta.go [hunk 6]
```diff
@@ -154,7 +155,7 @@ func resourceScalewayLbCertificateBetaRead(d *schema.ResourceData, m interface{}
 
 	_ = d.Set("name", res.Name)
 	_ = d.Set("common_name", res.CommonName)
-	_ = d.Set("subject_alternative_name", res.SubjectAlternativeName)
+	_ = d.Set("subject_alternative_name", strings.Join(res.SubjectAlternativeName, " "))
```
Comment (jerome-quere) [inline scaleway/resource_lb_certificate_beta.go:158]: subject_alternative_name sould be a list of string not a string

**Review by jerome-quere — CHANGES_REQUESTED** [review]

**Review by jerome-quere — APPROVED** [review]

**Pull request merged**